include/nuttx/sched.h

```diff
@@ -222,6 +222,14 @@
 
 #define get_current_mm()             (get_group_mm(nxsched_self()->group))
 
+/* Get task name from tcb */
+
+#if CONFIG_TASK_NAME_SIZE > 0
+#  define get_task_name(tcb)         ((tcb) ? (tcb)->name : "<null>")
```

Comment:
remove else branch and crash directly

Reply:
Sorry, I didn't get it. `get_task_name` is called everywhere even CONFIG_TASK_NAME_SIZE == 0

Reply:
What I mean is to remove the tcb check, and the caller must ensure that the tcb pointer is valid